### ISSUE TITLE
DT::datatable only appears once when using tabset?

### ISSUE THREAD

**Issue by ahdee** [atusy/minidown#73]
Hi I found a strange behavior, that if I try to use DT::datatable more than once then any table after the first does not appear. However this is only an issue when tabset is set to true?  Is there anything I can do to prevent this from happening. Here is reproducible example. 

````

---
title: "test"
output:
  minidown::mini_document:
    toc: true
    toc_float: true
    toc_highlight: true
    framework: water
    theme: light
    tabset: true
---



# Toptics  {.tabset}

## subheader1 {.tabset}

### Oranges
  * test

```{r fig=TRUE,fig.width=15, fig.height=3, echo=TRUE, include=TRUE  } 
DT::datatable( head ( Orange) )
print('')
```


### tooth growth 

  * test

```{r fig=TRUE,fig.width=5, fig.height=3, echo=TRUE, include=TRUE } 
DT::datatable( head ( ToothGrowth) )
```

### iris

```{r fig=TRUE,fig.width=5, fig.height=3, echo=TRUE, include=TRUE , results='asis' } 
 

 DT::datatable( head ( iris, 50 )  ,
          caption = '', rownames = F, 
          class = 'cell-border stripe', 
         # filter = 'top',
          extensions = 'Buttons',
          fillContainer = FALSE, 
          options = list(pageLength = 6, 
                         autoWidth = F,
                         dom = 'Bfrtip',
                         buttons = c('copy', 'csv', 'excel', 'pdf', 'print'), 
                         scrollX = TRUE, 
            selection="multiple"
          )) 
  

  
```
````

**Comment by atusy** [comment]
Thank you for the report.
I reproduced the issue.
It is strange but table appears when changing window size...

On the other hand, the bug does not appear with **reactable** package.
This means something particular happens with **DT** package.

BTW, for the next time, please try minimize the example.
Also, you should wrap Rmd file with ` ```` ` instead of ` ``` ` to avoid conflict with inner code blocks.
I applied the latter fix on my side by editing your comment.

**Comment by atusy** [comment]
This should be fixed by PR #74.

Try `remotes::install_github('atusy/minidown')` after the PR is merged or try `remotes::install_github('atusy/minidown#74')`.

**Comment by ahdee** [comment]
@atusy works perfectly now thanks and thanks for the tip with the wrapper; I was confused as to how to get around this.
